server/asset/dcr/dcr.go

```diff
@@ -75,13 +75,16 @@ type dcrNode interface {
 	GetBlockVerbose(blockHash *chainhash.Hash, verboseTx bool) (*chainjson.GetBlockVerboseResult, error)
 	GetBlockHash(blockHeight int64) (*chainhash.Hash, error)
 	GetBestBlockHash() (*chainhash.Hash, error)
+	GetBlockChainInfo() (*chainjson.GetBlockChainInfoResult, error)
 }
 
 // Backend is an asset backend for Decred. It has methods for fetching output
 // information and subscribing to block updates. It maintains a cache of block
 // data for quick lookups. Backend implements asset.Backend, so provides
 // exported methods for DEX-related blockchain info.
 type Backend struct {
+	// net is the dex.Network.
+	net dex.Network
```

Comment:
I might be missing the usage, but it seems `net` is an extraneous change here and in btc.

Reply:
It was from when I was doing `Synced -> false` for `estimatesmartfee` errors on mainnet. Removing.